### PR TITLE
[ALLUXIO-1490] Add a command to output current leader master

### PR DESCRIPTION
**Pull request by saltylin**
https://alluxio.atlassian.net/browse/ALLUXIO-1490

### CODE REVIEW

**Comment by uronce-cc** [comment]
LGTM

File: docs/en/Command-Line-Interface.md [hunk 1]
```diff
@@ -187,6 +187,13 @@ For example, `getUsedBytes` can be used to monitor the health of your cluster.
 
 {% include Command-Line-Interface/getUsedBytes.md %}
 
+## leader
+The `leader` command returns the current Alluxio leader master host name.
```
Comment (aaudiber) [inline docs/en/Command-Line-Interface.md]: The `leader` command prints the current ...

File: docs/en/Command-Line-Interface.md [hunk 2]
```diff
@@ -187,6 +187,13 @@ For example, `getUsedBytes` can be used to monitor the health of your cluster.
...
+
+For example, `leader` can be used to print the current leader master host name.
```
Comment (aaudiber) [inline docs/en/Command-Line-Interface.md]: The example doesn't add anything new so we can omit it

**Comment by aaudiber** [comment]
Thanks for adding this feature @saltylin! Could you also add a `LeaderCommandTest` unit test?

**Comment by gpang** [comment]
Thanks for this feature!

This is a more higher level question but would it be useful to have a type of master info command, which could also print out other information like all masters, current leader, zookeeper nodes, etc?

File: docs/en/Command-Line-Interface.md (hunk 1 above)
Comment (saltylin) [inline docs/en/Command-Line-Interface.md]: Updated!

File: docs/en/Command-Line-Interface.md (hunk 2 above)
Comment (saltylin) [inline docs/en/Command-Line-Interface.md]: Updated!

**Comment by saltylin** [comment]
@aaudiber Thanks for your review. I've added the `LeaderCommandTest` unit test.

**Comment by saltylin** [comment]
@gpang I agree, and have created the jira ticket for this purpose. https://alluxio.atlassian.net/browse/ALLUXIO-1976

**Comment by gpang** [comment]
LGTM

**Comment by apc999** [comment]
LGTM. per @gpang 's suggestion, that can be followup PRs.

**Pull request merged**